ninjato-core/src/main/kotlin/com/agoda/ninjato/http/HttpClient.kt

```diff
@@ -40,7 +40,7 @@ abstract class HttpClient(
      * @param request library's request entity
      * @return library's response entity
      */
-    abstract fun execute(request: Request): Response
+    abstract suspend fun execute(request: Request): Response
```

Comment:
Changing the existing HttpClient to use `suspend` and wrap it with `runBlocking ` in non-suspend functions. This avoids duplication of this file